e2e_test/ddl/invalid_operation.slt

```diff
@@ -0,0 +1,394 @@
+# 1. drop non-existent relation
+
+statement error table not found
```

Comment:
Can we maintain these error messages without pain? 🥵

Reply:
What is the pain you mean?

Reply:
Do we have to manually update all these error messages if we change them in the future?

Reply:
Yes, but since it’s regexp match instead of full match, and it won’t be frequently changed, I think it won’t be a huge pain 🫣